templates/cis2-nft/cargo-generate.toml

```diff
@@ -1,5 +1,5 @@
 [template]
-cargo_generate_version = ">= 0.17.0, < 0.22.0"
+cargo_generate_version = ">= 0.17.0, < 0.23.0"
```

Comment:
Is there a reason why we don't just have the version set to "0.22.0" instead of this wide version?

Reply:
If people installed `cargo-generate` a while ago, their local version of `cargo-generate` is not up-to-date. However, they can still use our templates (meaning we don't want to force people to run a specific `cargo-generate` version locally just because our templates require them).

We introduced the templates when `cargo-generate` was at version `0.17.0` and tested from that version until the current version. Lower versions might also work, we just never tested them.

Reply:
Makes sense